### DISCUSSION TITLE
在每个页面增加编辑跳转

### DISCUSSION THREAD

**Discussion by Spike-Leung** [Pavinberg/emacs-book#4] [Proposal]
我看了一些页面，存在一些文字错误。
之前看到一些文档是有编辑按钮，直接跳转到对应的文档进行编辑的。
也许可以考虑在页面中也放一个，有人看到了想改的话就方便一些。

**Comment by Pavinberg** [comment]
这样会极大增加网站的复杂度，涉及互动和版本管理，GitHub Pages 本身也只是个静态网站难以支持。以及最核心的问题其实是，目前并没有那么多人想参与编辑🥺

**Reply by Spike-Leung** [reply]
哈哈，有空我把看到的文字错误改一下。

教程我看完了，找到了一些不错的包，thanks ~

我以为是只需要放一个编辑按钮，跳转到 Repo 上对应的 org 文件，然后想编辑的人自己 fork 去修改就好了。

没想到会这么麻烦~

**Reply by Pavinberg** [reply]
哦哦，你的意思是跳转到 GtiHub 再 fork 呀，那可以做。有时间可以安排一下。
